server/application/terminal.go

```diff
@@ -92,6 +93,7 @@ func (s *terminalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	podName := GetQueryValue(q, "pod")
 	container := GetQueryValue(q, "container")
 	app := GetQueryValue(q, "appName")
+	project := GetQueryValue(q, "projectName")
 	namespace := GetQueryValue(q, "namespace")
 	shell := GetQueryValue(q, "shell")
```

Comment:
Just looking at this code, I was wondering whether we would want to validate the values that we pull from the user's request here, e.g. by matching against regexp?

Reply:
Good catch. We have validation functions defined above. I thought we were calling them, but now looks like we're not. I'm gonna change that.